ietf/doc/views_doc.py

```diff
@@ -139,20 +139,17 @@ def render_document_top(request, doc, tab, name):
 def interesting_doc_relations(doc):
 
     if isinstance(doc, Document):
-        cls = RelatedDocument
```

Comment:
This is here for a reason?
You will report the wrong relations if you are looking at an older version of a document if you take this out.